TweetMate/src/main/java/com/ybsystem/tweetmate/models/entities/twitter/TwitterStatus.java

```diff
@@ -7,10 +7,13 @@
 import androidx.core.text.HtmlCompat;
 
 import com.ybsystem.tweetmate.application.TweetMateApp;
+import com.ybsystem.tweetmate.databases.PrefSystem;
 import com.ybsystem.tweetmate.models.entities.Entity;
 import com.ybsystem.tweetmate.databases.PrefAppearance;
 import com.ybsystem.tweetmate.databases.PrefTheme;
+import com.ybsystem.tweetmate.utils.ToastUtils;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'com.ybsystem.tweetmate.utils.ToastUtils'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://192.168.11.19:9000/coding_rules#rule_key=squid%3AUselessImportCheck)